pytest/lib/cluster.py

```diff
@@ -866,16 +867,75 @@ def init_cluster(num_nodes,
         len(node_dirs), num_nodes, num_observers)
 
     logger.info("Search for stdout and stderr in %s" % node_dirs)
+
     # apply config changes
     for i, node_dir in enumerate(node_dirs):
         apply_genesis_changes(node_dir, genesis_config_changes)
         overrides = client_config_changes.get(i)
         if overrides:
             apply_config_changes(node_dir, overrides)
 
+    # apply config changes for nodes marked as archival node.
+    # for now, we do this only for local nodes (eg. nayduck tests).
+    if is_local:
```

Comment:
removed is_local. my concern was the the operations here should not work for remote (since it does local filesystem operations). but anyways rest of the code for changing configs also do not check is_local. maybe I misinterpret is_local. @marcelo-gonzalez to confirm.